app/src/main/res/layout/activity_main.xml

```diff
@@ -43,4 +43,16 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/launchSignInView" />
 
+    <Button
+        android:id="@+id/launchPoiView"
```

Comment:
I suggest removing this button and integrating the poi view with the map. What do you think?

Reply:
Yes but then onPoiClick has to be tested I think, which is tricky